docusaurus/docs/React/custom-code-examples/connection-status.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: connection_status
+sidebar_position: 17
+title: Connection Status
+---
+
+import ConnectionStatus from '../assets/ConnectionStatus.png';
+
+This example will show you how to change the message text in the `ConnectionStatus` component. This status appears when there is connection issues with the Stream Chat API.
```

Comment:
'is a connection issue' or 'are connection issues'

Reply:
actually don't worry about it, i'll merge and make the slight change in my geolocation pr